.ci/settings/settings.py

```diff
@@ -1,4 +1,3 @@
 ALLOWED_EXPORT_PATHS = ["/tmp"]
-ORPHAN_PROTECTION_TIME = 0
```

Comment:
Are we sure this isn't going to break some tests?

Reply:
Sure? Well we put it in a loooong time ago as a stop gap. I don't think we do that in the repositories CI anymore.